driver/lib/sessions/ios.ts

```diff
@@ -35,10 +32,18 @@ export const getObservatoryWsUri = async (proxydriver) => {
   if (realDevice) {
     // @todo check if `brew install usbmuxd` is needed
     log.info(`Running on iOS real device, doing "iproxy" now`);
-    const cmd = `iproxy ${urlObject.port} ${urlObject.port} ${udid}`;
-    log.debug(cmd);
-    await execPromise(cmd);
-    log.info(`"iproxy" successfully`);
+    const args = [urlObject.port, urlObject.port, udid];
+    log.debug(`iproxy ${urlObject.port} ${urlObject.port} ${udid}`);
```

Comment:
Line 35: Add `-u` argument in args before `udid`
Line 36: Add `-u` argument in debug log before `${udid}`

Reply:
-u or without this can have a same result

Reply:
@shibupanda Thanks for the Pull Request! I tested this a bit. It seems -u cannot be ignored. I am not sure if you have a different version or something. Here are some outputs from iproxy:
```
iproxy --help
Usage: iproxy [OPTIONS] LOCAL_PORT:DEVICE_PORT [LOCAL_PORT2:DEVICE_PORT2 ...]

Proxy that binds local TCP ports to be forwarded to the specified ports on a usbmux device.

OPTIONS:
  -u, --udid UDID    target specific device by UDID
  -n, --network      connect to network device
  -l, --local        connect to USB device (default)
  -s, --source ADDR  source address for listening socket (default 127.0.0.1)
  -h, --help         prints usage information
  -d, --debug        increase debug level
  -v, --version      prints version information

Homepage:    <https://libimobiledevice.org>
Bug Reports: <https://github.com/libimobiledevice/libusbmuxd/issues>
$ iproxy 6234:6235 -u 00008020-001D0D1C0E88003A
Creating listening port 6234 for device port 6235
waiting for connection
^C
$ iproxy 6234 6235 -u 00008020-001D0D1C0E88003A
Creating listening port 6234 for device port 6235
waiting for connection
^C
$ iproxy 6234 6235 00008020-001D0D1C0E88003A
Invalid listen port specified in argument '6234'!
 which iproxy
/usr/local/bin/iproxy
$ ls -l /usr/local/bin/iproxy
lrwxr-xr-x  1 daniel.han  admin  37 Jan 15 20:48 /usr/local/bin/iproxy -> ../Cellar/libusbmuxd/2.0.2/bin/iproxy
```

Although the manual says it should be `iproxy <local port>:<device port> -u uuid`, the ":" can be ignored during my test. Can you double check?

Reply:
Let's add -u and test it once then will merge it.
Let me know if that will work for you.

Reply:
Preferably add `:` as well to increase the readability.

Reply:
I tested it with `-u` locally and it works fine.

Reply:
Ok lets add it and merge so it should not impact other.